src/configurator/validators/offline.js

```diff
@@ -165,9 +165,15 @@ export default (
   }
 
   // Disable offline for development if user defined offline config
-  if (config.dev && config.offlineConfig.appcache) {
+  if (config.dev && config.offline) {
```

Comment:
This doesn't disable SW in dev mode if user set `offline = true`

It should be `if(!config.dev && config.offline)`